internal/commands/network/modify.go

```diff
@@ -68,21 +77,75 @@ func (s *modifyCommand) ExecuteSingleArgument(exec commands.Executor, arg string
 	logline := exec.NewLogEntry(msg)
 
 	logline.StartedNow()
-	logline.SetMessage(fmt.Sprintf("%s: sending request", msg))
 
-	res, err := exec.Network().ModifyNetwork(&request.ModifyNetworkRequest{
-		UUID:       arg,
-		Name:       s.name,
-		Zone:       "", // TODO: should this be implemented?
-		Router:     s.router,
-		IPNetworks: networks,
-	})
-	if err != nil {
-		logline.SetMessage(ui.LiveLogEntryErrorColours.Sprintf("%s: failed (%v)", msg, err.Error()))
-		logline.SetDetails(err.Error(), "error: ")
-		return nil, err
+	var network *upcloud.Network
+	if s.name != "" || len(networks) > 0 {
+		// we want to update name and/or networks
+		logline.SetMessage(fmt.Sprintf("%s: sending modify request", msg))
+		res, err := exec.Network().ModifyNetwork(&request.ModifyNetworkRequest{
+			UUID:       arg,
+			Name:       s.name,
+			Zone:       "", // TODO: should this be implemented?
```

Comment:
this fields isn't even in the API spec

Reply:
I wonder if it's a leftover from an admin API? Anyhow, this TODO is a leftover from the very early refactoring days, so I'd still keep it there.

Reply:
ok